contrib/ansible/openshift/pbench_register.yml

```diff
@@ -0,0 +1,36 @@
+---
+- hosts: controller
+  user: root
+  vars:
+    - default_tools_interval: "{{ DEFAULT_TOOLS_INTERVAL|default('10') }}"
+    - ose_master_interval: "{{ OSE_MASTER_INTERVAL|default('10') }}"
+    - ose_node_interval: "{{ OSE_NODE_INTERVAL|default('10') }}"
+    - script_dir: "{{ SCRIPT_DIR|default('/tmp') }}"
+    - file: /tmp/file
+  tasks:
+    - name: clear file contents
+      file: path={{ file }} state=absent
+    - name: copy, set permissions on the register script
+      copy:
+        src: register.sh
+        dest: "{{ script_dir }}/register.sh"
+        owner: root
+        group: root
+        mode: 0755
+    - name: add masters to the file
+      shell: echo -e {{ item.1 }} master {{ item.0 }} >> {{ file }}
+      with_indexed_items:
+       - "{{ groups['masters'] }}"
+    - name: add nodes to the file
+      shell: echo -e {{ item.1 }} node {{ item.0 }} >> {{ file }}
+      with_indexed_items: 
+       - "{{ groups['nodes'][-1] }}" 
+       - "{{ groups['nodes'][-2] }}"
+    - name: add etcd nodes to the file
+      shell: echo -e {{ item.1 }} etcd {{ item.0 }} >> {{ file }}
+      with_indexed_items: "{{ groups['etcd'] }}"
+    - name: add lb to the file
+      shell: echo -e {{ item.1}} lb {{ item.0 }} >> {[ file }}
+      with_indexed_items: "{{ groups['lb'] }}"
+    - name: register tools
+      shell: /bin/sh "{{ script_dir }}/register.sh" {{ default_tools_interval }} {{ ose_master_interval }} {{ ose_node_interval }}
```

Comment:
Not sure you need to use `/bin/sh` here because you already made this executable above.